lib/assets/javascripts/utils/tabHelper.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-env browser */ // This allows us to reference 'window' below
```

Comment:
Perfect, if we start using window across multiple files we can transfer that to lib/assets/.eslintrc.json under env key like jasmine and jquery.